src/ServiceControl.Audit/Infrastructure/Settings/Settings.cs

```diff
@@ -20,6 +20,9 @@ public Settings(string serviceName = null)
                 ServiceName = DEFAULT_SERVICE_NAME;
             }
 
+            // Overwrite the service name if it is specified in ENVVAR, reg, or config file
+            ServiceName = SettingsReader<string>.Read("InternalQueueName", ServiceName);
+
             {
```

Comment:
I know this block was here previously but it struck me as odd. I wasn't aware that you could have random blocks in C#. Does it make sense to leave this? I'm ok either way, just new to me.